debug/index.html

```diff
@@ -58,7 +58,8 @@
     zoom: 12.5,
     center: [-77.01866, 38.888],
     style: 'mapbox://styles/mapbox/streets-v8',
-    hash: true
+    hash: true,
+    workerCount: 3
```

Comment:
I've been doing a little housecleaning on this debug page to keep it as simple as possible. My intent (we're not there yet) is for this page to be as simple as possible, making it an ideal place to build one-off testing rigs, and to put any persistent specialized setups on other pages (i.e. `video.html`)

tl;dr 👍 @mourner